### PR TITLE
 manage_statuses

### PR DESCRIPTION
**Pull request by nguyenvanthanh1906**
## Related Tickets:
- [#36639](https://edu-redmine.sun-asterisk.vn/issues/36639)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by nguyenvanthanh1906** [comment]
ready

File: resources/views/statuses/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@extends('layouts.app', ['page' => 'statuses'])
+
+@section('content')
+  <div class="container">
+      @include('common.notification')
+      <h2>{{ trans('statuses.statuseslist') }}</h2>
+      <a href="{{Route('statuses.create')}}" class="btn btn-primary float-right">{{ trans('statuses.newstatus')}}</a>
+      <table class="table">
+        <thead>
```
Comment (quoctd-0607) [inline resources/views/statuses/index.blade.php]: fix hết indent nhé e

**Comment by nguyenvanthanh1906** [comment]
ready

**Comment by quoctd-0607** [comment]
commented
conflict code rồi e

**Comment by nguyenvanthanh1906** [comment]
ready

**Pull request merged**